Product/Production/Common/CONNECTCoreLib/src/main/java/gov/hhs/fha/nhinc/callback/openSAML/HOKSAMLAssertionBuilder.java

```diff
@@ -133,9 +133,11 @@ public Element build(final CallbackProperties properties) throws Exception {
             final PrivateKey privateKey = certificateManager.getDefaultPrivateKey();
             // sign the message
             signedAssertion = sign(assertion, certificate, privateKey, publicKey);
-        } catch (final Exception ex) {
+        } catch (final SAMLAssertionBuilderException ex) {
             LOG.error("Unable to create HOK Assertion: {}", ex.getLocalizedMessage());
             throw ex;
+        } catch (final Exception ex) {
```

Comment:
Why we suppress this generic exception and throw SAMLAssertionBuilderException above?  Should we remove the generic exception here?